docs/Guide.md

```diff
@@ -16,9 +20,9 @@ npx ignite-cli@latest new PizzaApp
 
 It'll walk you through several questions.
 
-Once it's up and running, you can use the Ignite CLI to [generate](./Generators.md) components, screens, MST models, and more.
+Once it's up and running, you can use the Ignite CLI to [generate](./concept-docs/Generators.md) components, screens, MST models, and more.
```

Comment:
is the docs postfix necessary? 

`docs/concept-docs` vs `docs/concept`

Reply:
Not really, I was just following @markrickert's Reactotron PR to keep things consistent. If you want to agree on a structure we can make sure to follow it!

Reply:
I don't think `/docs/WHATEVER-docs/` is necessary. If anything, organizationally, we should prefix the folders with their position in the sidebar imho

Reply:
I like that for the most part, until you have to insert one and then have to go changing all the links. But I guess we could increment by 10 or something

Reply:
https://docusaurus.io/docs/sidebar/autogenerated#using-number-prefixes

> By default, Docusaurus will remove the number prefix from the doc id, title, label, and URL paths.

However, we'd still have to update the links in the markdown files. I'll do some work on organizing things tongiht